pkgs/applications/audio/tonelib-gfx/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, dpkg
+, lib
+, autoPatchelfHook
+, fetchurl
+, gtk3
+, glib
+, desktop-file-utils
+, alsaLib
+, libjack2
+, harfbuzz
+, fribidi
+, pango
+, freetype
+}:
+
+stdenv.mkDerivation rec {
+  pname = "tonelib-gfx";
+  version = "4.6.6";
+
+  src = fetchurl {
+    url = "https://www.tonelib.net/download/0509/ToneLib-GFX-amd64.deb";
```

Comment:
Does the 0509 mean this download is versioned or is the file later replaced when a new version comes out?

Reply:
Yes, it is for this version